src/activity_actor.cpp

```diff
@@ -355,7 +355,7 @@ void autodrive_activity_actor::start( player_activity &act, Character &who )
 
 void autodrive_activity_actor::do_turn( player_activity &act, Character &who )
 {
-    if( who.in_vehicle && who.controlling_vehicle && player_vehicle->is_autodriving &&
+    if( who.in_vehicle && who.controlling_vehicle && player_vehicle && player_vehicle->is_autodriving &&
```

Comment:
There is another dereference of `player_vehicle` in line 369 below.

Reply:
Yes but it doesn't matter because if its null then the check on 358 catches it and it goes to the else on 365 where it returns before hitting that.